app/services/sign_in/token_serializer.rb

```diff
@@ -78,15 +76,15 @@ def token_json_payload
     end
 
     def cookie_authentication_client?
-      Constants::ClientConfig::COOKIE_AUTH.include?(client_id)
+      client_config.cookie_auth?
     end
 
     def api_authentication_client?
-      Constants::ClientConfig::API_AUTH.include?(client_id)
+      client_config.cookie_auth? == false
```

Comment:
Seeing this makes me think we should prob have an `api_auth?` function in client_config.rb. Also, I think we should have an `api_auth` field. Even though it should be mutually exclusive with cookie_auth, I think it'll be more clear if it's there